include/deal.II/base/task_result.h

```diff
@@ -357,6 +419,36 @@ namespace Threads
   }
 
 
+  template <typename T>
+  template <typename Callable>
+  void
+  TaskResult<T>::try_emplace_task(const Callable &creator) const
+    DEAL_II_CXX20_REQUIRES((std::is_invocable_r_v<T, Callable>))
+  {
+    // If the result is already available, simply return it.
```

Comment:
What do you mean by "it" here?

Reply:
Copy-paste mistake. The function returns `void`.